.github/workflows/verification-tests.yml

```diff
@@ -0,0 +1,86 @@
+---
+name: Verification Tests
+
+env:
+  IMAGE_NAME: job-server
+  PUBLIC_IMAGE_NAME: ghcr.io/opensafely-core/job-server
+  REGISTRY: ghcr.io
+  SSH_AUTH_SOCK: /tmp/agent.sock
+
+on:
+  pull_request:
+  workflow_dispatch:
+  schedule:
+    - cron:  "47 0 * * *"
+
+jobs:
+  test:
+    runs-on: ubuntu-22.04
+
+    services:
+      postgres:
+        image: postgres:13
+        env:
+          POSTGRES_USER: user
+          POSTGRES_PASSWORD: password
+          POSTGRES_DB: jobserver
+        ports:
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: "opensafely-core/setup-action@v1"
+        with:
+          python-version: "3.12"
+          install-just: true
+
+      - name: Install Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: ".node-version"
+          cache: "npm"
+          cache-dependency-path: package-lock.json
+
+      - name: Install node_modules
```

Comment:
I don't think we need to run any set up for the assets or to run the js tests, because I don't think our verification tests require them.